firmware/common/usb.c

```diff
@@ -28,6 +28,11 @@
 #include "usb_queue.h"
 #include "usb_standard_request.h"
 
+// avoid multiple #define declarations
+#ifndef LPC43XX_M4
+	#define LPC43XX_M4
+#endif
```

Comment:
This define should be supplied by the build system. In `firmware/hackrf-common.cmake` we have:
```cmake
SET(CFLAGS_M4 "-std=gnu99 ${CFLAGS_COMMON} ${CPUFLAGS_M4} -DLPC43XX_M4")
SET(CXXFLAGS_M4 "-std=gnu++0x ${CFLAGS_COMMON} ${CPUFLAGS_M4} -DLPC43XX_M4")
```
I realise you're integrating this code into your own build system rather than using ours, but I think the correct fix is going to be to do the same thing in your build system rather than setting it in this source file.

It needs to be set when compiling _all_ code that runs on the M4 and uses libopencm3, not just this one file.

Reply:
Thanks for your comment. I removed the define.